dbfit-java/build.gradle

```diff
@@ -62,7 +72,7 @@ subprojects {
         testCompile "org.hamcrest:java-hamcrest:2.0.0.0"
         testCompile "commons-io:commons-io:2.4"
         testRuntime 'com.dbdeploy:dbdeploy-ant:3.0M3'
-        integrationTestCompile "dummy:fitnesse-standalone:${fitNesseVersion}@jar"
+        integrationTestCompile "org.fitnesse:fitnesse:${fitNesseVersion}:standalone@jar"
```

Comment:
> I wonder if we can have some of the changes not related to maven central publishing as separate PR (dependencies configuration refactoring, etc.)

Certainly. I'm not sure what's not directly related to publishing though. Any ideas welcomed.